data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -186,7 +186,7 @@ The function `cat_hi`, which resulted from the partial application of `cat`, beh
 
 ### Type Variables and Higher-Order Functions
 
-A function may expect a function as a parameter, which is called a _higher-order_ function. A well-known example of higher-order function is `List.map`. Here is how it can be used:
+A function may expect a function as argument, which is called a _higher-order_ function. A well-known example of higher-order function is `List.map`. Here is how it can be used:
```

Comment:
```suggestion
A function may expect a function as a parameter, which is called a _higher-order_ function. A well-known example of higher-order function is `List.map`. Here is how it can be used:
```

It is when applying `map (fun x -> x * x)` that `fun x -> x * x` is higher-order argument.